Homework_12/Task_1.py

```diff
@@ -0,0 +1,24 @@
+from sympy import *
+x = Symbol("x")
+fx = x**3 - 50*x
+gx = -x**4 + 88*x**2 - 241
+fgx = fx - gx
+
+roots = solve(fgx)
+roots
+
+cross_dict = {}
+c = 1 
+for z in roots:
+    cross_dict[c] = {"x": round(N(z),5), "y": round(N(-1*z**4 - 88*z**2-241),5)}
+    c+=1
+cross_dict
```

Comment:
Чтобы вывести красиво:
```suggestion
for key, value in cross_dict.items():
    print(key, value)
```